contracts/orderbook/bZxInterfaces/IBZX.sol

```diff
@@ -0,0 +1,59 @@
+pragma solidity ^0.8.4;
+abstract contract IBZX{
```

Comment:
since you're trying to merge it. please use IBZX interface that is in top interface folder

Reply:
right now we have in the interface 
```
pragma solidity >=0.5.0 <=0.8.4;
```
wonder why it won't compile?

Reply:
we would remove a lot of duplicate interfaces if we somehow manage to compile things